src/app/public/modules/checkbox/checkbox.component.ts

```diff
@@ -102,6 +138,7 @@ export class SkyCheckboxComponent implements ControlValueAccessor, OnInit {
    * When you set this property to `true`, the component adds `aria-required` and `required`
    * attributes to the input element so that forms display an invalid state until the input element
    * is complete. This property accepts a `boolean` value.
```

Comment:
We should delete `This property accepts a boolean value.` because we don't call that out in the updated docs since the type is indicated in the first column.

Reply:
Done